test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -14,9 +14,9 @@ tags:
 
 1. Login via `oc` as a user with **cluster-admin** role (kubeadmin):
 
-   ```
-   oc login --token=<TOKEN> --server=https://api.<CLUSTER_NAME>.s1.devshift.org:6443
-   ```
```

Comment:
Only a small thing but can we revert the indentation change here? Had done this purposely so the code block aligns with the text